wp-remember-the-galleries/wp-remember-the-galleries.php

```diff
@@ -275,33 +314,29 @@ static function save_gallery() {
 					$captions[$id] = sanitize_text_field( $image['caption'] );
 				}
 			}
-		}
+			else {
+				self::json_error( 'invalid-input' );
```

Comment:
It's not suuuper clear, but `json_error()` calls `wp_send_json_error()` which calls `wp_send_json()` which will print some JSON and `die`.